### ISSUE TITLE
Problem with column and media query

### ISSUE THREAD

**Issue by AyeshaIkhlas21** [AyeshaIkhlas21/modularity-mindfulness#1]
@thatkidmadri

<img width="1680" alt="Screen Shot 2021-10-01 at 9 36 27 PM" src="https://user-images.githubusercontent.com/77422853/135699847-96e3fa99-195b-46a1-8466-e79a01739827.png">

**Comment by thatkidmadrid** [comment]
In order for me to test your code on my end you need to be sure to push your latest changes. As it is now, your gh-pages output is not behaving as the screenshot you shared above

**Comment by thatkidmadrid** [comment]
Some advice I can give for this activity is to remember that you should start designing for the small screen first, then add media queries to style larger screen sizes.

So, definitely try to nail the 320px screenshot first. You're close, but this will make spotting what you need to fix for the larger screens much clearer

**Comment by AyeshaIkhlas21** [comment]
Ok. 
So should I add 3 different media query or only one?  - Just curious

**Comment by thatkidmadrid** [comment]
If you look at the 3 screenshots, there only really seems to be a clear change in layout between the 320px and the 608. because you don't need a media query for the smallest as it is the default styling, this should tell you that you need at least 1 media query to achieve what's shown in the screenshots

**Comment by AyeshaIkhlas21** [comment]
Ok. I am trying. I will fix. 
If I have a trouble with it, can I text you here this weekend?

**Comment by thatkidmadrid** [comment]
I'll try my best to keep an eye on this issue thread

**Comment by AyeshaIkhlas21** [comment]
Hi, 
I am trying to fix things like margin and padding. But 3 screenshots are really different. Can you check my code if anything mistake or missing? Let me know please.


![image](https://user-images.githubusercontent.com/77422853/135899183-4c447922-f30e-4205-932f-dabec98a58b4.png)

**Comment by thatkidmadrid** [comment]
Hi Ayesha,

The way to approach this activity was to first style and match the small screen first and make sure that matches as close as possible. This shows how styling larger screens and working in reverse can be really difficult and complicated.

So, I will provide feedback on the small screen styling.

- The layout of the header is incorrect for the small screen. You may find flexbox useful for positioning the h1 properly, as well as the items in the nav.
- notice in the reference there seems to be some padding to keep the contents of the header, main and footer away from the edges of the screen
- In the small screen, you still have the 3 column layout for the articles
- the values in the [activity overview](https://learn-the-web.algonquindesign.ca/courses/web-design-2/week-03/#modularity-mindfulness) will get the reference to match within 1%

After you nail down the styling for the small screen it will be much easier to comment on the styling of the other sizes

**Comment by AyeshaIkhlas21** [comment]
I am sorry. I don't completely understand this assignment. I am deaf and have comprehension  issues. Will you be able to help me understand this assignment? This assignment is very difficult for me.

**Comment by thatkidmadrid** [comment]
This activity will make use of CSS Grids, Flexbox, and media queries

It will also require you to select the correct html tags to recreate the reference.

The key to this activity is to recognize there is a header, a main, and a footer area and to use the proper tags to create that structure.

Also, that flexbox can be used to layout the headings that consist of the japanese characters and the english words, the navigation links at the top, and the items inside each of the components at the bottom (the "viewing", "gravel,", and "symbolism" image, header, text, and link)

In class I went over an example of how the `<i>` tag is being used
```
<h1><i lang="ja">枯山水</i> <span>Rock Gardens</span></h1>
```
`<i>` is being used similarly to a `<span>` to be able to style sections of text. I'll go over more about its use in class. In this activity it's used to wrap around all of the japanese text, as well as it allows us to style them to stack on top of the English text using `display: flex` + `flex-direction: column` on the header.

It is also important to recognize that there is a grid being used when the screen size exceeds a certain width - This is where the media query comes in. The simplest grid is a 3-column grid just in the section containing the "viewing", "gravel,", and "symbolism" chunks. Flexbox properties that we went over earlier in the semester is the simplest way to get these to align properly.

There is a more complicated way to get this layout by making a 3x3 grid that covers everything that I'll quickly go over in class, but it is not the easiest solution.

**Comment by thatkidmadrid** [comment]
The deadline for submission for everyone in the class will be the end of the night tonight, but with the LOA we'll see how far you get and we'll discuss a timeline for when you can submit it

**Comment by AyeshaIkhlas21** [comment]
So this assignment is due tonight?

**Comment by thatkidmadrid** [comment]
yes, but I am looking at where you are now, and we can extend if you need it

**Comment by AyeshaIkhlas21** [comment]
I am messed up!

![Screen Shot 2021-10-06 at 2 59 52 PM](https://user-images.githubusercontent.com/77422853/136266371-0df22145-0a9d-424f-a975-01ff5bec802d.png)

**Comment by thatkidmadrid** [comment]
Please provide more information next time so I know what you're referring to and what you're stuck on- also, Unless there's something specific you want to point out, including a screenshot of the code and website like this is not necessary; I can pull up your repo and code through GitHub :)

I'm guessing you're stuck on the middle section, and getting it to align to the center horizontally.

I see you've used the `auto` value for your margin which is good, but you also set a margin in one of your media queries that is overriding this margin value. Think of things defined inside your media queries as overriding what you've defined by default. This is why working backwards from large-screen to small-screen is very difficult.

**Comment by AyeshaIkhlas21** [comment]
Hi, 
Last one- How do this box stay row ? Flexbox or align?

![image](https://user-images.githubusercontent.com/77422853/136618990-7cd8ae69-b19d-450c-a768-2eb76f5d5dce.png)

**Comment by thatkidmadrid** [comment]
To get the links to align to the link at the bottom of the tallest column, you can remove the display flex from the `<p>` and apply it to the `<article>`s instead. Remember to set the flex-direction to column, and then when you set a `flex-grow: 1` on the `<p>` it will allow it to grow to fill the extra space of the shorter columns and push the links to the bottom - aligning with the link of the tallest column

That will get them to align with each other, then you will have to change the other spacing values to match the reference

**Comment by AyeshaIkhlas21** [comment]
I fixed! I submitted. 
Thank you for help :)
